js/google_ima.js

```diff
@@ -524,13 +526,33 @@ require("../html5-common/js/utils/utils.js");
       });
 
       /**
-       * Called by the ad manager controller.  Ad Manager Controller lets the module know that an ad should play now.
-       * @public
+       * debouncing the case when user forwarded ahead and 2 or more ads requests simultaneosly issued
+       * @see https://jira.corp.ooyala.com/browse/PLAYER-3832
+       * @private
        * @method GoogleIMA#playAd
-       * @param {object} ad The ad to play from the timeline.
+       * @param {object} ad The ad to play from the timeline
+       * @returns {void}
        */
+      let debouncerId = null;
       this.playAd = function(amcAdPod)
       {
+        if (debouncerId) {
+          _resetAdsState();
+        }
+        debouncerId = setTimeout(() => {
+          debouncerId = null;
+          playAd(amcAdPod);
+        }, DEBOUNCE_TIMEOUT);
+      }
+
+      /**
+       * Called by the ad manager controller.  Ad Manager Controller lets the module know that an ad should play now.
+       * @private
+       * @method GoogleIMA#playAd
+       * @param {object} ad The ad to play from the timeline.
+       */
+      let playAd = (amcAdPod) => {
```

Comment:
please use the old style of function declaration

Reply:
@pilievOoyala oh, Peter! I am so pity to hear that we have to use old style! Why? 
ES6 is good practice is not it?
... And we use it already in the plugins, skin and mjolnir ...

Reply:
I also agree we should be using ES6. @pilievOoyala what is the reason we should use the old style of function declaration?

Reply:
You guys have convinced me. I see the advantages of the new style and not needing to do a bind.